types/tuple.hh

```diff
@@ -141,6 +141,24 @@ public:
         boost::range::for_each(range, put);
         return ret;
     }
+    template <typename RangeOf_bytes_opt>  // also accepts bytes_view_opt
+    static managed_bytes build_value_fragmented(RangeOf_bytes_opt&& range) {
+        size_t size = 0;
```

Comment:
I'm surprised it doesn't accept fragmented input, too. But I'm all for incremental development, so no hurry.

Reply:
It is replaced with a version which takes fragmented input in one of the next patches.

Reply:
Good.